source/direct_debit/index.html.md.erb

```diff
@@ -5,285 +5,253 @@ weight: 80
-    </tr>
-    <tr>
-    <td nowrap><code> PENDING </code></td>
-    <td nowrap><code> ACTIVE </code></td>
-    <td> The issuing bank has confirmed that funding source (for example, Direct Debit mandate) is valid. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> PENDING </code></td>
-    <td nowrap><code> FAILED </code></td>
-    <td> The issuing bank has rejected the funding source. Possible reasons: incorrect bank account details, bank account requires multiple signatories to set up a Direct Debit. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> PENDING </code></td>
-    <td nowrap><code> CANCELLED </code></td>
-    <td> Cancelled by the payer. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> ACTIVE </code></td>
-    <td nowrap><code> CANCELLED </code></td>
-    <td> Cancelled by the payer. </td>
-    </tr>
-  </table>
-</div>
+- be longer than 255 characters
+- contain URLs
 
-## One-off payments using Direct Debit
+The format is variable.
 
-To take a single payment using Direct Debit, you must set up a Direct Debit mandate.
+#### return_url
 
-1. Create an API key linked to your Direct Debit service.
+`return_url` is an HTTPS URL on your site. GOV.UK Pay will direct your user to this url when they've finished setting up the mandate.
 
-1. Follow the instructions in the [“Payment flow” section](/payment_flow) of this documentation.
+The URL should not be JSON-encoded, because backslashes are invalid characters in the API.
 
-The same API call is used for single payments in all contexts, including
-Direct Debit:
 
-POST /v1/payments
+### 2. Direct the user to GOV.UK Pay
 
+When the API responds, you need to:
 
-You cannot take further payments from users using this method without creating
-a new payment. This requires users to re-enter their account details.
+- direct the user to the `next_url` from the API response
+- store the `mandate_id` from the API response, so you can use it to collect payments against the mandate later
 
-## Variable payments using Direct Debit
+Refer to the <a
+href="https://govukpay-api-browser.cloudapps.digital/#newpayment"
+target="blank">GOV.UK Pay API browser</a> or the <a href="https://github.com/alphagov/pay-publicapi/blob/master/swagger/swagger.json" target="blank">Swagger file</a> for more
+information on the API response.
 
-You can collect payments of variable amounts at any time by using a variable agreement.
-For example you may want to collect:
+### 3. The user provides their bank account details
 
-* money each time a service is provided
+When your service redirects the user to `next_url`, they’ll see a **Set up a Direct Debit** page. The user enters their bank account details and selects **Continue**.
 
-* payments where the amounts vary according to service usage
+GOV.UK Pay checks with GoCardless that the user's account details are valid.
```

Comment:
I share Katie’s concern that a reader may infer that the details are definitely valid and that the mandate set-up won’t fail. Can we do something weaker like, “GOV.UK Pay checks with GoCardless that the user’s account number and sort code match a real bank account”?

Reply:
Sounds good! I'll add this.